core/tests/test_utils.py

```diff
@@ -319,9 +367,18 @@ def set_moderators(self, moderator_usernames):
             config_domain.MODERATOR_USERNAMES, moderator_usernames)
 
     def get_current_logged_in_user_id(self):
+        """Returns the USER_ID from environment variable USER_ID"""
```

Comment:
Gets the user_id of the current logged-in user.

(Also add "Returns" part of docstring, and include type information. See other documented files for examples.)